build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -7,9 +7,14 @@ spec:
   selector:
     app: flow-aggregator
   ports:
-  - port: 4739
+  - name: ipfix-udp
+    port: 4739
     protocol: UDP
     targetPort: 4739
+  - name: ipfix-tcp
+    port: 4739
+    protocol: TCP
+    targetPort: 4739
 ---
 apiVersion: apps/v1
 kind: Deployment
```

Comment:
we should add code to create namespace since both service and deployment are under flow-aggregator namespace

Reply:
Thanks, addressed.